docs/proposals/job_controller.md

```diff
@@ -0,0 +1,192 @@
+# Job Controller
+
+## Abstract
+A proposal for implementing a new controller - Job controller, which will be responsible
+for managing pods that require to run:
+* once on demand or at a specified point in time
+* repeatedly at a specified point in time.
+
+Several exiting issues were already created regarding that particular subject:
+* Distributed CRON jobs [#2156](https://github.com/GoogleCloudPlatform/kubernetes/issues/2156)
+* Job Controller [#1624](https://github.com/GoogleCloudPlatform/kubernetes/issues/1624)
+
+
+## Use Cases
+1. Ability to have a controller that will create a series of run-once pod(s).
+1. Ability to implement a MapReduce model.
+1. Ability to have a controller that will have an option to specify triggers on pod’s success/failure.
+1. Ability to specify number of restarts upon job failure.
+1. Ability to specify maximum running time for a job.
+1. Ability to create a repetitive job, eg. database backup, sending emails.
+1. Ability to create one time job to be run at a specified time, eg. system analysis.
+1. Ability to create a chain of jobs dependent one on another.
```

Comment:
Agreed, the entire topic of triggers is something TBD in the longer term, still I'd like to at least mention the need for such mechanism in here.